operator/option/config.go

```diff
@@ -132,6 +132,10 @@ const (
 	// API to fill out the instancetype to adapter limit mapping.
 	UpdateEC2AdapterLimitViaAPI = "update-ec2-apdater-limit-via-api"
 
+	// UpdateEC2AdapterLimitViaAPIv2 configures the operator to use the EC2
+	// API to fill out the instancetype to adapter limit mapping.
+	UpdateEC2AdapterLimitViaAPIv2 = "update-ec2-adapter-limit-via-api"
+
```

Comment:
I'd rename this const and re-use the name for the "new option" to better reflect that the misspelledoption is deprecated and so we don't forget to remove it in 1.10:
```suggestion
	UpdateEC2AdapterLimitViaAPIDeprecated = "update-ec2-apdater-limit-via-api"

	// UpdateEC2AdapterLimitViaAPI configures the operator to use the EC2
	// API to fill out the instancetype to adapter limit mapping.
	UpdateEC2AdapterLimitViaAPI = "update-ec2-adapter-limit-via-api"

```